matic/contracts/WildcardSteward_matic_v2.sol

```diff
@@ -1244,4 +1244,50 @@ contract WildcardSteward_matic_v1 is Initializable, BasicMetaTransaction {
 
         assetToken.transferFrom(_currentOwner, address(this), tokenId);
     }
+
+    function transferWithDeposit(
+        address to,
+        uint256 tokenId,
+        uint256 newPrice,
+        uint256 tokensDeposit
+    )
```

Comment:
Possible attack vector. Lets not worry but keep this in mind.  

Its a stretch, but you can send a token with the min deposit (of 1 week) at quite a high price to another user, and after 1 week this token may be quite a strain on all the users deposit causing them to foreclose

Reply:
Basically you are sending someone a debt obligation

Reply:
I guess the new user can immediately withdraw the deposit they received?

Reply:
Yeah, exactly, bad that I'm only looking at this now :grapes:

Reply:
I think it is fine, if it is enough to drain the users deposit after the week, then it is a large contribution to wildlife conservation too!